draft-ietf-dnsop-dns-capture-format-00.md

```diff
@@ -1390,56 +1387,64 @@ this should be taken into consideration.
 A capture of sample data from a root instance was used for the
 comparison. The input file was 661.87Mb. The following table shows the
 formatted size and size after compression (both in Mb), together with
-the RSS (in kb) and user time (seconds) taken by the
-compression.
+the task resident set size (RSS) in kb and the user time in seconds
+taken by the compression.
 
-Format|Fmt size|Comp.|Size|RSS|User time
-:-----|-------:|:----|---:|--:|--------:
+Format|File size|Comp.|Comp. size|RSS|User time
```

Comment:
It would be nice. But it has a rather negative effect on the table layout which I can't counteract because Markdown is too limited.